app/javascript/components/common/TrackIcon.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 
-export function TrackIcon({ track, width = 100 }) {
+export function TrackIcon({ track }) {
   return (
     <img
-      style={{ width: width }}
+      className="c-track-icon"
```

Comment:
We can add a default width/height to this component if we want to in the CSS. I've not for now though.